staging/src/k8s.io/cri-client/pkg/errors/errors.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package exec
+package errors
 
 import (
 	"time"
```

Comment:
there is a TODO in this file suggesting we can start using grpc errors: `TODO: this error type can likely be removed when we support CRI errors.` I *think* it is a leftover after dockershim. Did you by chance have a chance to look at how expensive will it be to eliminate the need of this type before the migration?

Reply:
Sure, following-up in https://github.com/kubernetes/kubernetes/pull/124739